CHANGELOG.adoc

```diff
@@ -156,6 +158,7 @@ These color palettes are accessible in studio projects but are not visible in th
 - [ADR-104] Add support for Help Expressions in Form widgets
 - [ADR-105] Add custom widget to edit EMF references
 - [ADR-106] Add the support for the palette with react flow
+- [ADR-107] - Improve frontend selection support
```

Comment:
- Should be moved to 2023.12 (and remove the " -" which is not present in the others.
- We're now at ADR 122, this would be (at least) ADR 123.

Reply:
Done